S21/rebelDi/tests/diana/WordToLowerCaseTest.java

```diff
@@ -0,0 +1,72 @@
+package diana;
+
+import org.junit.Test;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+import static org.junit.Assert.*;
+
+public class WordToLowerCaseTest {
+
+    /**
+     * This is the test method for WordToLowerCase class,
+     * that tests the result of changeWordToLowerCase()
+     */
+    @Test
+    public void changeWordToLowerCase() throws IOException {
+        List<String[]> wordsToProcess = openTSV();
```

Comment:
I would like to know how openTSV() works.

Reply:
It is a function later in the code. It just opens up the file, reads it element by element and adds each element to the array of the row. I can move the function to the top of the class, so that it would be visible first thing first?